libraries/freertos_plus/standard/freertos_plus_tcp/source/FreeRTOS_TCP_IP.c

```diff
@@ -1660,14 +1682,14 @@ static NetworkBufferDescriptor_t *prvTCPBufferResize( const FreeRTOS_Socket_t *p
 	int32_t lDataLen, UBaseType_t uxOptionsLength )
 {
 NetworkBufferDescriptor_t *pxReturn;
-int32_t lNeeded;
+uint32_t lNeeded;
```

Comment:
The prefix `l` in `lNeeded` stands for long, `uint32_t` would become `ul`.
But I propose to make it `size_t uxNeeded`.
Furtheron stop using the macro `FreeRTOS_max_uint32()`, and in stead:
~~~c
    uxNeeded = ipNUMERIC_CAST( size_t, ipSIZE_OF_ETH_HEADER + uxIPHeaderSizeSocket( pxSocket ) +
                    ipSIZE_OF_TCP_HEADER + uxOptionsLength ) + lDataLen;
    if( uxNeeded < sizeof( pxSocket->u.xTCP.xPacket.u.ucLastPacket ) )
    {
        uxNeeded = sizeof( pxSocket->u.xTCP.xPacket.u.ucLastPacket );
    }
~~~
And also, you can substitute `( size_t ) lNeeded` with just `uxNeeded`.
~~~